src/lib/text-utils.ts

```diff
@@ -43,8 +43,8 @@ export function buildTrimmedText(text: string, length: number): TrimmedTextResul
 
   let prefixLength = Math.floor(length / 2)
   const suffixLength = length - prefixLength - 1
-  const prefix = text.substr(0, prefixLength)
-  const suffix = text.substr(text.length - suffixLength, suffixLength)
+  const prefix = text.substring(0, prefixLength)
+  const suffix = text.substring(text.length - suffixLength, text.length)
```

Comment:
`substr` is now deprecated: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr